docs/TROUBLESHOOTING.md

```diff
@@ -99,6 +99,19 @@ This section includes how to debug a failed restore. For more specific issues re
  
 <hr style="height:1px;border:none;color:#333;">
 
+<h3 align="center">Restore Errors and Warnings<a id="creds"></a></h3>
+
+Restore errors and warnings are shown in the `velero describe` output in three groups:
+1. Velero: Warnings or errors that relate to the overall restore workflow but not to a particular resource will be shown here.
+2. Cluster: Warnings or errors that relate to particular cluster-scoped resources will be shown here
+3. Namespaces: Warnings or errors that relate to particular namespaced resources will be shown here, grouped by namespace
+
+One or more errors in any of the above categories will cause a Restore to be `PartiallyFailed` rather than `Completed`. Warnings will not cause a change to the completion status.
+
+For resource-specific errors ("Cluster" and "Namespaces" errors), the `restore describe --details` output should include the resource list which will list all resources that Velero succeeded in restoring. For any that errored out, check to see if the resource is actually in the cluster.
```

Comment:
@sseago The word "should" is ambiguous -- could be advice, could be "must" or "needs to" [Israelis often use 'should" when they mean this option], or "if everything works as expected." Use "recommend" for the first, "must" or  "needs to" for the second, and simply state as a fact for the third. 
I am reading "should" as "how it's supposed to work." If that's wrong, correct me on https://github.com/openshift/openshift-docs/pull/67803, which I've requested you to review.